backend/src/test/java/com/festago/auth/application/LoginServiceTest.java

```diff
@@ -0,0 +1,79 @@
+package com.festago.auth.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+
+import com.festago.auth.domain.SocialType;
+import com.festago.auth.domain.UserInfo;
+import com.festago.auth.dto.LoginMemberDto;
+import com.festago.domain.Member;
+import com.festago.domain.MemberRepository;
+import com.festago.support.MemberFixture;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator.ReplaceUnderscores;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@DisplayNameGeneration(ReplaceUnderscores.class)
+@SuppressWarnings("NonAsciiCharacters")
+@ExtendWith(MockitoExtension.class)
+class LoginServiceTest {
+
+    MemberRepository memberRepository;
+
+
+    LoginService loginService;
+
+    @BeforeEach
+    void setUp() {
+        memberRepository = mock(MemberRepository.class);
+        loginService = new LoginService(memberRepository);
+    }
```

Comment:
기존에는 `@BeforeEach`로 Mock을 초기화 시켜줬네요!
다음과 같이 하는건 어떠신가요?
```suggestion
@Mock
MemberRepository memberRepository;

@Mock
AuthProvider authProvider;

@InjectMocks
LoginService loginService;
```